### DISCUSSION TITLE
Just how to use this?

### DISCUSSION THREAD

**Discussion by vm6ej04** [ruflin/Elastica#2223] [General]
Hi... I don't mean to be rude, and it may sound extremely stupid.
I've downloaded, composed it, successfully hosted with Nginx as host server.
Just to realize, Index.php was blank.
I feel like it is a collection of pre coded php scripts to interact with elastic's API or something...
I imagined it would be something like Cerebro(which doesn't support Elastic 8.0^), just need to enter host IP, credential and voila~~
Apparently not@@?

**Comment by ruflin** [comment]
Hi, Elastica is an Elasticsearch client that needs to built into your php application. Here is a getting started: https://elastica.io/getting-started/
